gcp_variant_transforms/transforms/densify_variants_test.py

```diff
@@ -30,13 +30,25 @@
 class DensifyVariantsTest(unittest.TestCase):
   """Test cases for the ``DensifyVariants`` transform."""
 
-  def test_add_missing_calls(self):
-    transform = densify_variants.DensifyVariants()
-    variant = vcfio.Variant(calls=[vcfio.VariantCall(name='sample2')])
-    new_variant = transform._densify_variants(
-        variant, ['sample1', 'sample2', 'sample3'])
-    call_names = [call.name for call in new_variant.calls]
-    self.assertItemsEqual(call_names, ['sample1', 'sample2', 'sample3'])
+  def test_densify_variants_pipeline_no_calls(self):
+    call_names = ['sample1', 'sample2', 'sample3']
```

Comment:
nit: since call_names is now only used once (and are intentionally ignored), consider just hardcoding it inside `VariantCall(name=...)`. You can add a comment about why the calls are ignored too.